File: src/main/java/ru/dragonestia/apocalypse/item/TinClusterItem.java
package ru.dragonestia.apocalypse.item;

import cn.nukkit.item.Item;
import ru.dragonestia.expo.item.AdvItem;

public class TinClusterItem extends Item implements AdvItem {

    public TinClusterItem(Integer meta, int count) {
        super(ApocalypseID.TIN_CLUSTER, meta, count, "Tin cluster");
    }

    @Override
    public String getRuName() {
        return "Оловянная руда";
    }

    @Override
    public String getIcon() {
        return "textures/items/cluster/tin";
    }

}

File: src/main/java/ru/dragonestia/apocalypse/item/TinClusterItem.kt
package ru.dragonestia.apocalypse.item

import cn.nukkit.item.Item
import ru.dragonestia.expo.item.AdvItem

class TinClusterItem @JvmOverloads constructor(meta: Int? = 0, count: Int = 1) : Item(ApocalypseID.TIN_CLUSTER, 0, count, "Tin cluster"), AdvItem {

    override fun getRuName(): String = "Оловянная руда"

    override fun getIcon(): String = "textures/items/cluster/tin"

}